README.md

```diff
@@ -1,67 +1,51 @@
+- Enables applications to connect and perform firmware updates in a few lines of code.
+- Runs separately from your main application; it does not take over your main loop.
+- Provides LWM2M resources, variables that sync automatically through Pelion Device Management.
+- Helps users avoid doing blocking network operations in interrupt contexts, by automatically defering actions to a separate thread.
+- Provides end-to-end Greentea tests for Pelion Device Management.
 
-
-## Table of Contents
-
-1. [Device management for your Mbed OS application](#device-management-for-your-mbed-os-application)
-    * [Adding device management](#adding-device-management-feature-to-your-application)
-    * [Example applications and board specific examples](#example-applications)
-2. [Device management configuration](#device-management-configuration)
-    * [Context and terminology](#context-and-terminology)
-    * [Application configuration](#1-application-configuration)
-    * [Bootloader configuration](#2-bootloader-configuration)
-    * [Add the bootloader to your application](#3-add-the-bootloader-to-your-application)
-3. [Validation and testing](#validation-and-testing)
-    * [Test suites overview](#test-suites---basic)
-    * [Testing setup](#testing-setup)
-    * [Troubleshooting](#troubleshooting)
-4. [Known issues](#known-issues)
+This library makes it trivial to expose sensors, actuators and other variables to a cloud service. For a complete Pelion Device Management Client API, please see our [documentation](https://cloud.mbed.com/docs/current/mbed-cloud-client/index.html).
 
 ## Device management for your Mbed OS application
 
-First and foremost, it's important to note that not every device (microcontroller, module, board) is capable of running device management features. And while some hardware capabilities can be easily added or extended - like connectivity, storage, TRNG - others are impossible or inconvenient to extend (e.g. RAM/Flash).
+Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG, others are impossible or inconvenient to extend (for example, RAM or flash).
 
-Here's a list of minimum of requirements to add device management feature to your application:
-* RAM: 128K or more
-* Flash: 512K or more
-* Real Time Clock (RTC)
-* (optional but recommended) True Random Number Generator (TRNG)
-* A storage device - SDcard, SPI Flash, QSPI Flash, Data Flash
-* IP connectivity - Ethernet, Wi-Fi, Cellular, 6LoWPAN, Thread
+The minimum requirements to add device management feature to your application are:
 
-Additionally, the device and any additional complementary hardware components should be supported and provided in the latest releases of Mbed OS, or have support for the APIs provided in the latest releases of Mbed OS.
+- RAM: 128K or more.
```

Comment:
Will these numbers change in future releases, or do we expect them to remain the same?

Reply:
We do expect to stay the same.